src/Bridges/SymfonyConsole/CreateCommand.php

```diff
@@ -89,7 +141,92 @@ private function hasNumericSubdirectory($dir, & $found)
 				return TRUE;
 			}
 		}
+
 		return FALSE;
 	}
 
+
+	/**
+	 * @return string
+	 */
+	protected function getTypeArgDescription()
+	{
+		$options = [];
+		$groups = $this->config->getGroups();
+		usort($groups, function (Group $a, Group $b) {
+			return strcmp($a->name, $b->name);
+		});
+
+		foreach ($groups as $i => $group) {
+			for ($j = 1; $j < strlen($group->name); $j++) {
+				if (!isset($groups[$i + 1]) || strncmp($group->name, $groups[$i + 1]->name, $j) !== 0) {
+					$options[] = substr($group->name, 0, $j) . '(' . substr($group->name, $j) . ')';
+					break;
+				}
+			}
+		}
+
+		return implode(' or ', array_filter([
+			implode(', ', array_slice($options, 0, -1)),
+			array_slice($options, -1)[0],
```

Comment:
nn, array_slice  vrátí string[], [0] vrátí tedy poslední element; zkusím zlepšit čitelnost